compatibility_tests/common.py

```diff
@@ -0,0 +1,17 @@
+import asdf
+from asdf.versioning import supported_versions, AsdfVersion
+import numpy as np
+
+
+def generate_file(path, version):
+    if version not in supported_versions:
+        raise ValueError("ASDF Standard version {} is not supported by version {} of the asdf library".format(version, asdf.__version__))
+
+    af = asdf.AsdfFile({"array": np.ones((8, 16))}, version=version)
+    af.write_to(path)
+
+
+def assert_file_correct(path):
```

Comment:
You may want to add `__tracebackhide__ = True` to this assertion helper so that the code in the helper doesn't become part of the traceback when the test fails.  You just want to see the assertion failure rewrite in pytest.

Reply:
Yessir, thanks!